### ISSUE TITLE
Ad-hoc translation of article names having shared article

### ISSUE THREAD

**Issue by wvengen** [foodcoop-adam/foodsoft#147]
Vokomokum needs to have English article names in addition to the local language. We can do this by having a foodcoop-local article name, either in a new database column, or have a setting where (per article) the article name can be _not_ synchronised.

Work-in-progress at [foodcoops/feature/sync_skip_columns](https://github.com/foodcoops/foodsoft/tree/feature/sync_skip_columns)
- [x] Add sync_skip_columns to Article
- [x] Skip synchronisation on specified columns
- [x] Show checkbox in user-interface
- [ ] Use bootstrapicons/gyphicons with toggle [button](http://getbootstrap.com/2.3.2/javascript.html#buttons) instead of checkbox
- [x] Connect checkbox to sync_skip_columns
- [ ] Make sync screen enter old values for skipped fields in changed articles
- [ ] Test if functionality works (with updating sharedlists article)
- [ ] Update value from shared article when unchecking column using ajax / data attribute
- [ ] Remove `disabled` from input when unchecking column

**Comment by wvengen** [comment]
Being able to specify which fields are synchronised per article would be great. This would be stored in a new column containing the list of column names to synchronize. We'll implement this first in the user-interface for the name only.

**Comment by wvengen** [comment]
Temporary fixed by adding a foodcoop-note for shared articles. It would be nice to finish this, as it would bring a lot of clarity as to what the fields mean for shared articles.

**Comment by wvengen** [comment]
Since this is not really required anymore, I'd like to close this until the need comes up again.

**Comment by Cor-Jan** [comment]
I have no idea what it means. Can you explain?

**Comment by wvengen** [comment]
The approach we wanted to use for allowing translated articles names was to include the translated name in the note field. For shared articles, this meant that the article name needed to be not overwritten when synchronized.

To tackle this more generally, we started work to allow a foodcoop to choose which fields would be synchronized. But it proved a larger task than we had the time for.

In the meantime, I added a foodcoop-note in addition to the supplier's note for shared articles. This is now used to set the English name of a product, and shown in the article info popup in the member ordering screen (group_orders).

Since we have a solution for the title of this issue, I'd like to close this ticket. We can refer back to this issue when we'd like to allow foodcoops to select which article fields need to be synchronised, and which not.
